pyproject.toml

```diff
@@ -1,3 +1,35 @@
+[project]
+name = "s2geometry"
+description = "Computational geometry and spatial indexing on the sphere"
+authors = [
+    { name = "Dan Larkin-York", email = "dan@arangodb.com" },
```

Comment:
Could you either use the contents of `CONTRIBUTORS` here or leave it like in `setup.cfg` (in which case I'll fix it later)?